### PR TITLE
feat (cdsctl): use workflow run count

### PR DESCRIPTION
**Pull request by yesnault**
Signed-off-by: Yvonnick Esnault <yvonnick.esnault@corp.ovh.com>

### CODE REVIEW

**Comment by yesnault** [comment]
We have to limit GET queue size too. This have to be done with a limit on each project to avoid that a project with 1k jobs does not let another job from another project running. cf. #1679

File: engine/api/workflow/dao_node_run_job.go [hunk 1]
```diff
@@ -13,10 +13,11 @@ import (
 	"github.com/ovh/cds/engine/api/cache"
 	"github.com/ovh/cds/engine/api/group"
 	"github.com/ovh/cds/sdk"
+	"github.com/ovh/cds/sdk/log"
 )
 
-// LoadNodeJobRunQueue load all workflow_node_run_job accessible
-func LoadNodeJobRunQueue(db gorp.SqlExecutor, store cache.Store, groupsID []int64, since *time.Time, statuses ...string) ([]sdk.WorkflowNodeJobRun, error) {
+// return query, groupID, statuses, isSharedInfraGroup
+func queryLoadNodeJobRunQueue(isCount bool, groupsID []int64, since *time.Time, statuses ...string) (string, string, *time.Time, string, bool) {
```
Comment (fsamin) [inline engine/api/workflow/dao_node_run_job.go]: i don't like it, i would prefer two different functions event if there is a bit of code duplication 
- or may be groupID stuff should be in a function
- or the query should be in a constant

File: engine/api/api_routes.go [hunk 2]
```diff
@@ -288,6 +288,7 @@ func (api *API) InitRouter() {
 
 	//Workflow queue
 	r.Handle("/queue/workflows", r.GET(api.getWorkflowJobQueueHandler))
+	r.Handle("/queue/workflows/count", r.GET(api.countWorkflowJobQueueHandler))
```
Comment (fsamin) [inline engine/api/api_routes.go]: I miss unit tests :)

Comment (yesnault) [reply engine/api/api_routes.go]: done

**Pull request merged**